augur/curate/rename.py

```diff
@@ -0,0 +1,85 @@
+"""
+Renames fields / columns of the input data
+"""
+
+from typing import Iterable, Literal, Union, List, Tuple
+import argparse
+from augur.io.print import print_err
+from augur.errors import AugurError
+
+def register_parser(parent_subparsers):
+    parser = parent_subparsers.add_parser("rename",
+    parents = [parent_subparsers.shared_parser],
+    help = __doc__)
+
+    p = parser.add_argument_group(title="RENAME SPECIFIC OPTIONS")
+
+    p.add_argument("--field-map", nargs="+", default=[],
+        help="Rename fields/columns via '{old_field_name}={new_field_name}'. " +
+             "If the new field already exists, then the renaming of the old field will be skipped. " +
+             "Multiple entries with the same '{old_field_name}' will duplicate the field/column. " +
+             "Skips the field if the old field name is the same as the new field name (case-sensitive).")
+    p.add_argument("--force", action="store_true",
+        help="Force renaming of old field even if the new field already exists. " +
+             "Please keep in mind this will overwrite the value of the new field.")
+
+    return parser
+
+
+def parse_field_map(field_map_arg: List[str]) -> List[Tuple[str,str]]:
+    seen_new = set() # keep track of the new field names
+    
+    field_map = []
+    for field in field_map_arg:
+        old_name, new_name = field.split('=')
```

Comment:
worth stripping whitespace off `old_name` and `new_name` here?

Reply:
I had originally glossed over this thinking any sane IFS would take care of it for me, but of course it won't do this when the arguments are quoted. Fixed up in force push and a bunch of tests added here.